packages/client/src/users/__tests__/getUserAttribute.test.ts

```diff
@@ -0,0 +1,58 @@
+import { getUserAttribute } from '..';
+import { UserAttributesResponse } from '../types';
+import client from '../../helpers/client';
+import fixtures from '../__fixtures__/getUserAttribute.fixtures';
+import moxios from 'moxios';
+
+describe('getUserAttribute', () => {
+  const expectedConfig = undefined;
+  const userId = 123456;
+  const attributeId = '123456';
+  const spy = jest.spyOn(client, 'get');
+
+  beforeEach(() => {
+    moxios.install(client);
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => moxios.uninstall(client));
+
+  it('should handle a client request successfully', async () => {
+    const response: UserAttributesResponse = {
```

Comment:
I think these mock responses could be added to the general mock files that we have (`tests/__fixtures__/users/users.fixtures.ts`), and then import it from there, don't you think? 
You can check other areas to see how we've done it.

Also check other if this applies to the other files you changed 🙂

Reply:
done :D